Try AAA reporting tests again

emord opened the pull request.

emord committed:
Revert "Remove all test changes from this test"

This reverts commit f217dfdd2a9d6b39340a396641345094389ea626.

emord committed:
Revert "Remove aaa-data databse from docker's localsettings"

This reverts commit 578a098b5eb0c4ef9045449ea5c5ec95525ddde1.

emord committed:
Only run partitioned test for now

emord committed:
Remove Base from end of class name

emord committed:
Fix class name

emord committed:
Comment out the comparisons

emord committed:
Comment out test class

emord committed:
comment out paths

emord committed:
comment out setup

emord committed:
comment out teardown

emord committed:
Comment out constants

emord committed:
Coment out import from icds_reports

emord committed:
Coment out import from aaa

emord committed:
Comment everything

emord committed:
Remove file

emord committed:
Remove changes to icds reports base test

emord committed:
Remove changes to docker localsettings

emord committed:
r u 4 real travis?

emord committed:
u were not 4 real travis. try removing aaa-data database

emord committed:
Revert "u were not 4 real travis. try removing aaa-data database"

This reverts commit 02f86162455b2afa2339ac4cf515fc6f92ee0ed4.

emord commented:
it appears that for some reason adding the aaa-data database to django is causing the partitioning tests (but not others) to not successfully create some of the couch views. very confusing

emord committed:
Try different database name

millerdev commented:
I've run into similar things in the past as well. Let me know if you want another set of eyes to get it sorted.

emord commented:
@millerdev Thanks for the offer. I was able to work around this and introduced it in another PR (since I also added tests in that PR) https://github.com/dimagi/commcare-hq/pull/23389#issuecomment-467159153

For convience, the solution was to have aaa-data resolve to another database name. Right now both `default` and `icds-ucr` resolve to the database `commcarehq` when I added `aaa-data` the couch views started faililng. I have no idea why that was, but my best guess is a timing issue with our sync couch views and migration commands.

The fix was to have `aaa-data` resolve to `aaa_commcarehq` database
